book/src/qobject/cxxqtthread.md

```diff
@@ -7,15 +7,23 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # CxxQtThread
 
-`CxxQtThread<T>` is used for [threading](../concepts/threading.md) with QObject#s and allows you to queue events from a background thread to occur on the Qt event loop.
+`CxxQtThread<T>` is used for easy [threading](../concepts/threading.md) with QObjects.
+The QObjects generated by CXX-Qt are neither [`Sync`](https://doc.rust-lang.org/std/marker/trait.Sync.html) nor [`Send`](https://doc.rust-lang.org/std/marker/trait.Send.html).
+Therefore they may not be passed between threads.
 
-To access the `CxxQtThread<T>` use the `qt_thread(&self)` method on the [`CppObj`](./cpp_object.md) or on the C++ pointer of the QObject.
+`CxxQtThread<T>` is a handle to the Qt thread that the `qobject::T` lives on.
```

Comment:
"Handle" doesn't seem like quite the right word here; that makes it seem like it's similar to a [std::thread::JoinHandle](https://doc.rust-lang.org/std/thread/struct.JoinHandle.html). Maybe "helper for running Rust code in the Qt event loop"? That's a bit verbose though :/

Reply:
Hm, true, it's definitely not a JoinHandle.
Maybe just calling it "reference" to the Qt thread would work?

Reply:
"Reference" already has different meanings in Rust and C++, so I think that would be worse than "handle".

Reply:
Maybe just "`CxxQtThread<T>` is an object that allows you to queue closures safely onto the Qt thread that `qobject::T` lives on."

Reply:
Done